src/main/resources/static/css/style.css

```diff
@@ -1,12 +1,12 @@
 html {
     font-size: 18px;
-    -webkit-user-select: none;
-    -webkit-tap-highlight-color: transparent;
-    -webkit-touch-callout: none;
 }
 
 body {
     font-family: 'Source Sans Pro', sans-serif;
+    -webkit-user-select: none;
```

Comment:
Nedával bych.

Reply:
OK, já s tím vůbec nemám zkušenosti, nevím jak to s iOS funguje - tak to dám pryč. A ty další 2
-webkit-tap-highlight-color: transparent;
-webkit-touch-callout: none;
tam mám nechat?

Reply:
user-select by ti mělo zakázat výběr a kopírování textu. Když to máš takhle na body tagu, tak na celé stránce. webkit ale není jen na iPhonu, ale třeba i na Safari na Macu atd.

Callout nejspíš zakáže to kontextové menu, když nad odkazem podržíc prst. Na adnroidu je tam třeba možnost otevřít link v novém okně atd...

tap-highlight-color si myslím, můžeš nechat. Uvidíme co na to iOS experti ;)